packages/preview/casson-uom-thesis/0.1.0/README.md

```diff
@@ -0,0 +1,41 @@
+# Typst PhD thesis template for the University of Manchester
+
+Typst template based upon [The University of Manchester Presentation of Theses Policy](https://documents.manchester.ac.uk/display.aspx?DocID=7420) which relates to the examination of doctoral and MPhil degrees at The University of Manchester and applies to full-time and part-time postgraduate research students of the following degrees: Doctoral degrees: Doctor of Philosophy (PhD); Doctor of Medicine (MD) Doctor of Business Administration (DBA); Professional, Engineering and Enterprise Doctorates; Master of Philosophy (MPhil). This template has been checked to be compliant with the 2024 requirements. Responsibility for ensuring compliance with the University of Manchester Presentation of Theses Policy remains with the candidate.
+
+
+## Using the template on typst.app
+The template should ultimately be available on Typst Universe as casson_uom_thesis. Create an account at [Typst.app](https://typst.app/) and start a new project by clicking on Start from template and searching for casson_uom_thesis.
+
+Alternatively, you can download files from the template repository and upload them to your project folder. If doing this, in main.typ comment out
+
+  `#import "@preview/casson-uom-thesis:0.1.0": *`
+
+and instead uncomment
+
+  `//#import "casson-uom-thesis.typ": *`

```

Comment:
I think it would be cleaner to not have the commented line in the template by default, and suggest to replace the default import with the other one in this paragraph.